examples/CRISP/interfold.config.yaml

```
@@ -6,25 +6,25 @@ chains:
         address: "0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82"
-        deploy_block: 16
+        deploy_block: 21
       ciphernode_registry:
         address: "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707"
-        deploy_block: 8
+        deploy_block: 13
       bonding_registry:
         address: "0xa513E6E4b8f2a923D98304ec87F64353C4D5C853"
-        deploy_block: 9
+        deploy_block: 14
       bonding_registry:
```

YAMLlint (1.37.1)

[error] duplication of key "bonding_registry" in mapping

(key-duplicates)